Sources/PackageCollections/Model/Package.swift

```diff
@@ -226,3 +231,9 @@ extension Array where Element == PackageCollectionsModel.Package.Version {
             .first
     }
 }
+
+extension Dictionary where Key == ToolsVersion, Value == PackageCollectionsModel.Package.Version.Manifest {
+    public var `default`: Value? {
```

Comment:
@abertelrud @neonichu

Sometimes we need to choose just one manifest--ideally it should be `Package.swift`, but in the current model design there is no way to distinguish `Package.swift` from version-specific manifests. Here I am selecting the one with the latest tools version, which is true most of the time, but would it be better if there is a flag (e.g., `isDefault`) in `Manifest` instead?

Reply:
@abertelrud @neonichu Do you have any advice or suggestion on this?

Reply:
imo maybe a field that says "defaultToolsVersion" for the package, so there could only be one

Reply:
Thanks. Added `defaultToolsVersion` https://github.com/apple/swift-package-manager/pull/3226/commits/2aa760a84eb8b83b459442133bf375147a1cace7